ci/test_wheel.sh

```diff
@@ -15,7 +15,7 @@ rapids-logger "Download Wheel"
 RAPIDS_PY_WHEEL_NAME="numba_cuda" rapids-download-wheels-from-s3 ./dist/
 
 rapids-logger "Install wheel"
-python -m pip install $(echo ./dist/numba_cuda*.whl)
+python -m pip install $(echo ./dist/numba-cuda*.whl)
```

Comment:
Same as above, I think this needs to stay underscored.

Reply:
Thanks, undone as above.